### ISSUE TITLE
authority com.termux.documents not allowed

### ISSUE THREAD

**Issue by john-peterson** [AndroidIDEOfficial/AndroidIDE#1058]
### What happened?

https://github.com/termux/termux-packages/issues/7437

I did not read anything. It should work acode is editing inside termux. Why can't I build projects inside termux?

REPRODOCE

clone https://github.com/HardcodedCat/termux-monet

Open folder in aide through termux pipe. This message is shown for one second and NOTHING HAPPENS

Workaround fixes help? It's kind of the hello world that aide can build and open termux. Should be the first thing anyone test. Strange that I am the first testing anything obvious 

TLDR

The hello world of aide open termux and build   fails.  Facepalm

**Comment by john-peterson** [comment]
CLONE IT INSIDE AIDE

Which is a good thing

I'm getting tired. Closing
